src/VisualStudio/Core/Def/ServicesVSResources.resx

```diff
@@ -699,4 +699,7 @@ Additional information: {1}</value>
   <data name="Prefer_predefined_type" xml:space="preserve">
     <value>Prefer predefined type</value>
   </data>
+  <data name="Connection_to_remote_host_has_been_lost_some_features_might_stop_working_or_start_working_in_proc_do_you_want_to_recover_remote_host" xml:space="preserve">
+    <value>Connection to remote host has been lost. some features might stop working or start working in proc. do you want to recover remote host?</value>
```

Comment:
This isn't a resource shown to the user is it?

This is far too complicated, and users will not know what they shoudl do here.  We should either:

1) Tell them something went wrong, and that they need to restart VS.
2) Handle this gracefully (reporting something to watson) and start the service back up again.

Users should not be in control of if things run in/out of proc.  If they do we open ourselves up to a testing matrix nightmare where we have to write our features to work in either context and we have to test all the permutations of htings running in/out of proc.